src/parser/jobs/whm/modules/Swiftcast.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {t, Trans, Plural} from '@lingui/macro'
+
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+import {getDataBy} from 'data/getDataBy'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+import {ActionLink} from 'components/ui/DbLink'
+
+const MISSED_SWIFTCASTS_SEVERITIES = {
+	1: SEVERITY.MAJOR,
+}
+
+export default class Swiftcast extends BuffWindowModule {
+	static displayOrder = DISPLAY_ORDER.SWIFTCAST
+	static handle = 'swiftcast'
+	static title = t('whm.swiftcast.title')`Swiftcast Actions`
```

Comment:
Just a minor nit, and I only really noticed once it went live on SCH, but we might want to change this to something like 'Swiftcast Usage'

With the table column header, you see 'Swiftcast Usage' in two places really close to one another.

Perhaps change the column header instead? I dunno, just offering my two cents.

Reply:
Swiftcast Usage for the title is more in line with the language in other modules around xiva imo